src/pages/Auth/Login.js

```diff
@@ -91,10 +92,12 @@ export function Login() {
             Login with Test Credentials
           </div>
 
-          <div className="auth-secondary-btn text-center">
-            Create New Account{" "}
-            <i className="fa fa-chevron-right" aria-hidden="true"></i>
-          </div>
+          <Link to="/signup">
+            <div className="auth-secondary-btn text-center">
+              Create New Account{" "}
```

Comment:
suggestion: remove all the `{" "}` from your code, use padding or `&nbsp` if extra space is needed